beacon-chain/node/p2p_config.go

```diff
@@ -44,6 +49,32 @@ func configureP2P(ctx *cli.Context) (*p2p.Server, error) {
 		staticPeers = append(staticPeers, peers...)
 	}
 
+	if ctx.GlobalBool(cmd.Hobbits.Name) {
+		baseDir := ctx.GlobalString(cmd.DataDirFlag.Name)
+		dbPath := path.Join(baseDir, beaconChainDBName)
+		if ctx.GlobalBool(cmd.ClearDB.Name) {
+			if err := db.ClearDB(dbPath); err != nil {
+				return nil, err
+			}
+		}
+
+		hobbitsDB, err := db.NewDB(dbPath)
+		if err != nil {
+			return nil, err
+		}
+
+		s := hobbits.Hobbits(ctx.GlobalString(cmd.P2PHost.Name), ctx.GlobalInt(cmd.P2PPort.Name), staticPeers, hobbitsDB)
+
+		logrus.Debug("")
+		logrus.Trace("constructed a hobbits node...")
+
+		s.Start()
```

Comment:
Do not call start. It is called automatically as part of the service registry.

Reply:
+1